src/mlo_dir_conv.cpp

```diff
@@ -149,7 +149,12 @@ static auto GetWindogradSolvers()
     return miopen::solver::SolverContainer<miopen::solver::ConvBinWinograd3x3U,
                                            miopen::solver::ConvBinWinogradRxSf3x2,
                                            miopen::solver::ConvBinWinogradRxSf2x3,
-                                           miopen::solver::ConvBinWinogradRxS>{};
+                                           miopen::solver::ConvBinWinogradRxS,
+                                           miopen::solver::ConvMPBidirectWinograd<2, 3>,
```

Comment:
Did you see any case where F(2,3) shows better performance?

Reply:
So, you want to save only  F(5,3) and F(6,3)?